plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/WaitUtil.java

```diff
@@ -0,0 +1,55 @@
+package org.jboss.tools.openshift.reddeer.utils;
+
+import org.jboss.reddeer.common.condition.WaitCondition;
+import org.jboss.reddeer.common.wait.AbstractWait;
+import org.jboss.reddeer.common.wait.TimePeriod;
+
```

Comment:
This has to be implemented on RedDeer side adding constructors to wait methods allowing users to define WAIT_SLEEP period. Your solution has these issues:
* it's very dirty and WAIT_SLEEP variable cannot be renamed within RedDeer but is not part of API son never will be deprecated just changed. I know this is not very probable
* to much of code just to accomplish setting of some variable
* WAIT_SLEEP is static so is changed for all instances of AbstractWait and if will not be set back (e.g. in case test will fail sooner than it's reset back or user just forgot to set it back) it will stay changed for all following tests